src/ontology/cl-edit.owl

```diff
@@ -27172,6 +27174,22 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023006 "nuclear bag2 intraf
 AnnotationAssertion(rdfs:label obo:CL_4023006 "static nuclear bag fiber")
 SubClassOf(obo:CL_4023006 obo:CL_4023004)
 
+# Class: obo:CL_4023007 (static gamma motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:139469") Annotation(rdfs:comment "PMID:13933877") Annotation(rdfs:comment "WikipediaVersion:Gamma_motor_neuron&oldid=906640465") obo:IAO_0000115 obo:CL_4023007 "A gamma motor neuron that innervates static nuclear bag fibers (bag2 fibers) and increase their firing, in response to an increase in magnitude of change in length, and controls the static sensitivity of the stretch reflex.")
```

Comment:
Tried to follow the resource Paola suggested to write the definition, hopefully it's better now.